### PR TITLE
Add new Lab 13 Udica

### PR DESCRIPTION
**Pull request by wrabcak**
Lab udica is about generating SELinux security policy for custom container. 

@lkerner , Lab still needs some grammar review and maybe highlight some important parts. Please take a look, I'm ready for review. 
We'll need one more system for udica (RHEL-8.1)
Ansible playbook to preconfigure system is also read, verification playbooks is missing.

### CODE REVIEW

File: 2020Labs/RHELSecurity/ansible/udica/udica.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    authorized_key:
+      user: root
+      state: present
+      key: "{{ lookup('file', 'id_rsa.pub') }}"
+
+  - name: Set authorized key taken from file for adminlab
+    authorized_key:
+      user: auditlab
+      state: present
+      key: "{{ lookup('file', 'id_rsa.pub') }}"
```
Comment (bachradsusi) [inline 2020Labs/RHELSecurity/ansible/udica/udica.yml]: Drop both authorized_key's - for root and for user which is later auditlab

Comment (lkerner) [reply 2020Labs/RHELSecurity/ansible/udica/udica.yml]: @wrabcak : can you update the udica.yml to do what @bachradsusi suggested above and submit a new PR? Also, I noticed that this PR has a conflict we need to resolve. Can you look at that too?

**Comment by lkerner** [comment]
@wrabcak : this has a conflict with the session recording lab exercise #12 ... can you review?

**Comment by wrabcak** [comment]
I deleted my fork after this PR was created. If I want to fix conflict + comments from Petr, I need to create new PR.

**Pull request closed without merging**